plugins/check_curl.c

```diff
@@ -378,8 +378,12 @@ void
 handle_curl_option_return_code (CURLcode res, const char* option)
 {
   if (res != CURLE_OK) {
-    snprintf (msg, DEFAULT_BUFFER_SIZE, _("Error while setting cURL option '%s': cURL returned %d - %s"),
```

Comment:
This is just argument rearangements, I cannot find a reason, why this should have generated a compiler
warning before (if so, which compiiler warning is it fixing for clang and for which flag?).

This is a coding style issue: do we prefer compacter code or split arguments per line?

Reply:
this is most like not a compiler warning, but a styling thing I did to actually be able to read it